src/capi/impl/capi_sframe.cpp

```diff
@@ -229,20 +232,341 @@ EXPORT void tc_sframe_random_split(const tc_sframe* sf, double fraction,
+  ERROR_HANDLE_START();
+
+  return new_tc_sframe(sf->value.unstack(column, new_column_name));
+
+  ERROR_HANDLE_END(error, NULL);
+}
+
+EXPORT tc_sframe* tc_sframe_unstack_vector(
+    const tc_sframe* sf, const tc_flex_list* columns, const char* new_column_name, tc_error** error){
+  ERROR_HANDLE_START();
+
+  std::vector<std::string> columns_transform;
+
+  for (const turi::flexible_type& i : columns->value){
+    if(i.get_type() != turi::flex_type_enum::STRING){
+      set_error(error, "Columns is not of type str");
+      return NULL;
+    }
+
+    columns_transform.push_back(i.get<turi::flex_string>());
+  }
+
+  return new_tc_sframe(sf->value.unstack(columns_transform, new_column_name));
+
+  ERROR_HANDLE_END(error, NULL);
+}
+
+
+EXPORT tc_sframe* tc_sframe_unique(const tc_sframe* sf, tc_error** error){
+  ERROR_HANDLE_START();
+
+  return new_tc_sframe(sf->value.unique());
+
+  ERROR_HANDLE_END(error, NULL);
+}
+
+EXPORT tc_sframe* tc_sframe_sort_single_column(const tc_sframe* sf, const char* column, bool ascending, tc_error** error){
+  ERROR_HANDLE_START();
+
+  return new_tc_sframe(sf->value.sort(column, ascending));
+
+  ERROR_HANDLE_END(error, NULL);
+}
+
+EXPORT tc_sframe* tc_sframe_dropna(const tc_sframe* sf, const tc_flex_list* columns, const char* how, tc_error** error){
+  ERROR_HANDLE_START();
+
+  std::vector<std::string> columns_transform;
+
+  for (const turi::flexible_type& i : columns->value){
+    if(i.get_type() != turi::flex_type_enum::STRING){
+      set_error(error, "Columns is not of type str");
+      return NULL;
+    }
+
+    columns_transform.push_back(i.get<turi::flex_string>());
+  }
+
+  return new_tc_sframe(sf->value.dropna(columns_transform, how));
+
+  ERROR_HANDLE_END(error, NULL);
+}
+
+EXPORT tc_sframe* tc_sframe_sort_multiple_columns(const tc_sframe* sf, const tc_flex_list* columns, bool ascending, tc_error** error){
+  ERROR_HANDLE_START();
+
+  std::vector<std::string> columns_transform;
+
+  for (const turi::flexible_type& i : columns->value){
+    if(i.get_type() != turi::flex_type_enum::STRING){
+      set_error(error, "Columns is not of type str");
+      return NULL;
+    }
+
+    columns_transform.push_back(i.get<turi::flex_string>());
+  }
+
+  return new_tc_sframe(sf->value.sort(columns_transform, ascending));
+
+  ERROR_HANDLE_END(error, NULL);
+}
+
+
+EXPORT tc_sframe* tc_sframe_slice(const tc_sframe* sf, const uint64_t start, const uint64_t end, tc_error** error){
+  ERROR_HANDLE_START();
+
+  return new_tc_sframe(sf->value[{static_cast<long long>(start), static_cast<long long>(end)}]);
+
+  ERROR_HANDLE_END(error, NULL);
+}
+
+EXPORT tc_sframe* tc_sframe_slice_stride(const tc_sframe* sf, const uint64_t start, const uint64_t end, const uint64_t stride, tc_error** error){
+  ERROR_HANDLE_START();
+
+  return new_tc_sframe(sf->value[{static_cast<long long>(start), static_cast<long long>(end), static_cast<long long>(stride)}]);
```

Comment:
long long can be weird -- what's the specific type in the sf->value signature?  Let's do that explicitly.